examples/advanced/src/example.tsx

```diff
@@ -564,16 +564,34 @@ export default class Example extends React.Component<
                 this.state.modeConfig.turfOptions.units) ||
               'kilometers'
             }
-            onChange={(event) =>
-              this.setState({ modeConfig: { turfOptions: { units: event.target.value } } })
-            }
+            onChange={(event) => {
+              const modeConfig = {
+                ...this.state.modeConfig,
+                turfOptions: { units: event.target.value },
+              };
+              this.setState({ modeConfig });
+            }}
           >
             <option value="kilometers">kilometers</option>
             <option value="miles">miles</option>
             <option value="degrees">degrees</option>
             <option value="radians">radians</option>
           </select>
         </ToolboxControl>
+        <ToolboxTitle>Multipoint</ToolboxTitle>
+        <ToolboxControl>
```

Comment:
This is an excellent enhancement to this mode! I think we can get rid of the multipoint configuration option and just make it always use multipoint mode.

Reply:
I've changed measure tool behavior to multipoint by default